api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/QualityImprovement2020Queries.java

```diff
@@ -1225,4 +1225,83 @@ public static CohortDefinition getMQ11DenB4(
+      int yesConcept,
+      int breastfeedingConcept) {
+    SqlCohortDefinition sqlCohortDefinition = new SqlCohortDefinition();
+    sqlCohortDefinition.setName("Cat11 B5");
+    sqlCohortDefinition.addParameter(new Parameter("startDate", "startDate", Date.class));
+    sqlCohortDefinition.addParameter(new Parameter("endDate", "endDate", Date.class));
+    sqlCohortDefinition.addParameter(
+        new Parameter("revisionEndDate", "revisionEndDate", Location.class));
+    sqlCohortDefinition.addParameter(new Parameter("location", "location", Location.class));
+
+    Map<String, Integer> map = new HashMap<>();
+    map.put("6", adultoSeguimentoEncounterType);
+    map.put("856", hivViralLoadConcept);
+    map.put("1065", yesConcept);
+    map.put("6332", breastfeedingConcept);
+
+    String query =
+        " SELECT p.patient_id "
+            + " FROM patient p "
+            + " INNER JOIN ( "
+            + "             SELECT p.patient_id, MIN(e.encounter_datetime) as first_carga_viral  "
+            + "             FROM patient p  "
+            + "             INNER JOIN encounter e ON p.patient_id = e.patient_id  "
+            + "             INNER JOIN obs o ON e.encounter_id = o.encounter_id  "
+            + "             WHERE  p.voided = 0   "
+            + "                    AND e.voided = 0   "
+            + "                    AND o.voided = 0  "
+            + "                    AND e.encounter_type = ${6}    "
+            + "                    AND o.concept_id = ${856}   "
+            + "                    AND o.value_numeric > 1000  "
+            + "                    AND e.encounter_datetime BETWEEN :startDate AND :endDate   "
+            + "                    AND e.location_id = :location   "
+            + "             GROUP  BY p.patient_id    "
+            + "           ) AS lab ON lab.patient_id = p.patient_id  "
+            + " INNER JOIN (  "
+            + "             SELECT p.patient_id, e.encounter_datetime AS lactante  "
+            + "             FROM patient p   "
+            + "             INNER JOIN encounter e ON e.patient_id = p.patient_id   "
+            + "             INNER JOIN obs o ON e.encounter_id = o.encounter_id   "
+            + "             INNER JOIN person per ON per.person_id= p.patient_id  "
+            + "             WHERE   p.voided = 0   "
+            + "                     AND e.voided = 0   "
+            + "                     AND o.voided = 0   "
+            + "                     AND e.encounter_type = ${6}    "
+            + "                     AND o.concept_id = ${6332}    "
+            + "                     AND o.value_coded = ${1065}    "
+            + "                     AND e.encounter_datetime BETWEEN :startDate AND :endDate   "
+            + "                     AND e.location_id = :location    "
+            + "                     AND per.gender = 'F'      "
+            + "             GROUP BY p.patient_id  "
```

Comment:
@rupansana12 this group by will give you some wrong results
@Edgencio @helderjosue

Reply:
@psmatsinhe  sure, do you recommend removing it?